docs/running-tests/webkitgtk_minibrowser.md

```diff
@@ -0,0 +1,32 @@
+# WebKitGTK MiniBrowser
```

Comment:
Good stuff!

Reply:
@foolip Does this PR include Epiphany Technology Preview within WebKitGTK MiniBrowser?

Reply:
No, this PR is unrelated to Epiphany.
For running tests with Epiphany, you need to use the browser "epiphany" for "wpt run"